Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Ceph Dashboard Monitoring
```

Comment:
Or if there is overlap with the ceph-monitoring.md, maybe this content should be a new section in that doc.

Reply:
My first preference would be to keep this document separate, because it is long, with lot of things, and it is something not mandatory, so probably is adding complexity to the Ceph Dashboard document. However, I think that to include a link to this document in the original Ceph dashboard is interesting.

The OCP environment only needs Prometheus Operator to work, so seems sensible not to add in the Prometheus installation doc  all the stuff about the thing that need the dashboard in a pure k8s environment, but not in an OCP environment where all the elements are preinstalled.